README.md

```diff
@@ -1,92 +1,40 @@
-    <li><strong>positionOffset: </strong> The position offset that the tooltip keeps from the targetObject (default: 20)</li>
-    <li><strong>backgroundColor: </strong> The color of the background (default: 0x000000)</li>
-    <li><strong>strokeColor: </strong> The color of tooltip stroke (default: 0xffffff)</li>
-    <li><strong>strokeWeight: </strong> The line weight of the tooltip stroke (default: 2)</li>
-    <li><strong>customBackground: </strong> If the tooltip will use a custom background (default: false)</li>
-    <li><strong>position: </strong> The position of the tooltip based on the targetObject (default: top, options: top, bottom, left, right)</li>
-    <li><strong>enableCursor: </strong> Enables the hand-cursor over the target object when hovered (default: false)</li>
-    <li><strong>textStyle: </strong> Declares styles for the simple text element (default: {
-            fontSize: 12,
-            fill: "#ffffff",
-            stroke: "#1e1e1e",
-            strokeThickness: 1,
-            wordWrap: true,
-            wordWrapWidth: 200,
-            lineSpacing: -2
-        }</li>
-    <li><strong>onHoverCallback: </strong> Calls a function when hover occurs (default: fn)</li>
-    <li><strong>onOutCallback: </strong> Calls a function when hover ends (default: fn)</li>
-    <li><strong>fixedToCamera: </strong> Pins the tooltip on the camera and moves with it (x,y are now camera offset) (default: false)</li>
-</ul>
+Added options:
+- Rounded corners
+  - roundedCorderRadius
+- font styles
+  - fontSize
+  - fontStroke
+  - fontFill
+- roundedCorderRadius (always show tips. control with methods to show and hide)
 
-<strong>API Functions</strong>
-
-<ul>
-    <li><strong>printOptions: </strong> Prints the options specified in the constructor on the console</li>
-    <li><strong>updatePosition: </strong> Change the x, y of the tooltip</li>
-    <li><strong>destroy: </strong> Destroys the tooltip and all its children</li>
-    <li><strong>hideTooltip: </strong> Hide the tooltip</li>
-    <li><strong>showTooltip: </strong> Show the tooltip</li>
-</ul>
-
-<i>
-</i>
-
-<hr>
-
-<strong>Buy me a coffee or tea!</strong> <br>
-<a href="https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=JCFPKZJ7Y23JJ&lc=GR&item_name=NetGfx%2ecom&currency_code=EUR&bn=PP%2dDonationsBF%3abtn_donate_SM%2egif%3aNonHosted"><img src="https://www.paypalobjects.com/webstatic/en_US/btn/btn_donate_92x26.png"/></a>
-
-
-<hr>
-
->The TODO list is diminising!
-
->Please let me know if you come across some bug or have something to contribute
+Please see the source code at : https://github.com/BeFiveINFO/Phasetips/blob/master/Phasetips.js
 
+Example:
 
+```
+var tooltip_enter_name = new Phasetips(Game, {
+	targetObject: Register.spriteInstances.name_input_text_field_backpanel,
+	context: "Please enter your nickname to start game.",
+	fontSize: "24px Josefin Sans",
```

Comment:
according to this `_fontSize = _options.fontSize || 12;` its a number. to user string declaration you need to use `font` instead